sys-fs/squashfs-tools/files/squashfs-tools-4.3-extmatch.patch

```diff
@@ -0,0 +1,39 @@
+--- a/squashfs4.3/squashfs-tools/action.c
```

Comment:
this patch needs documentation.  please see https://dev.gentoo.org/~vapier/clean-patches.

rather than copy & paste the same workaround to many files, can't you find a common header file to stick it into ?